### ISSUE TITLE
Passing --exclude warnings into flake8

### ISSUE THREAD

**Issue by psychemedia** [mattijn/pycodestyle_magic#12]
I've recently been exploring this magic as an implicit study support tool for students learning Python coding using Jupyter notebooks, importing additional `flake8` packages to enhance the range of error messages provided including `pep8-naming` and `flake8-docstrings`.

One of the things have noticed about the docstrings linter is that in a notebook cell it throws a `D100` error calling out missing docstrings for modules. I want to call out missing docstrings in functions but do not require comments about a function. Typically this would require calling something like:

`flake --ignore D100 filename` 

or a comma separate list of errors  for ignoring multiple errors.

In `pycodestyle_magic`, it would be useful to be able to run the `%flake8_on` command with a similar set of excluding items, such as:

`%flake8_on --exclude X123,Y456`

**Comment by mattijn** [comment]
In my understanding is it not possible to have parameters for a magic function.

But @bsmoliak has made this https://github.com/mattijn/pycodestyle_magic/pull/9 before in which he updated the flake8 magic to use the `extend_ignore`. So if you have a local flake8 project configuration file then it should be possible to add there the codes which you want to ignore.

From the tests I distilled this code in which this could be tested:
```python
from pycodestyle_magic import flake8
import os
cell = '''a=1\n'''
path = os.path.join(os.getcwd(), '.flake8')
with open(path, "w") as fp:
    fp.write('[flake8]\nignore = E225\n')
    fp.flush()
    flake8(None, cell)
os.remove(path)
```

If I run this code in a notebook (jupyter-lab) than it works fine after I restarted the kernel, but upon re-running the cell it doesn't work anymore. I've no idea why

**Comment by mattijn** [comment]
'It works fine' means: there are no `flake8` errors/warnings as output
'It doesn't work anymore': there is a `flake8` error, but I expect none.

**Comment by psychemedia** [comment]
Ah, that's helpful.

Re: passing in parameters, magics can be created to pass arguments. I did a hacky magic [here](https://github.com/psychemedia/ipython_magic_folium/blob/master/folium_magic/folium_magic.py) that included lots of parameters. One  argument used CSV reader to parse comma separated values ([example](https://github.com/psychemedia/ipython_magic_folium/blob/master/folium_magic/folium_magic.py#L55)); [I don't know if there's a better way of doing that within `argparse`?!]

Recent IPython (v6+) also looks like it has some additional built in support for setting up arguments using `argparse` via decorators: https://ipython.readthedocs.io/en/stable/api/generated/IPython.core.magic_arguments.html

**Comment by mattijn** [comment]
Thanks for the comment with the suggestion of the `magic_arguments`! 

Now I could add this request as in https://github.com/mattijn/pycodestyle_magic/pull/17
I've added the `--ignore` option (and `--max_line_length`)

So one can do:
```
%flake8_on --ignore E225,E265
``` 
to ignore the errors `E225` and `E265`

(available in version >= `0.5`)
